clarity/contracts/key-token/key-wbtc-79760-usda.clar

```diff
@@ -1,4 +1,5 @@
 (impl-trait .trait-yield-token.yield-token-trait) 
+(impl-trait .trait-ownable.ownable-trait)
```

Comment:
As an aside, note that since `contract-caller` is used in the guards, there can be a situation where the owner is unable to invoke certain functions indirectly; i.e. via a different contract using `(as-contract (contract-call? ...))`.

Reply:
Yep, agreed.